docs/configuration/unit_testing_rules.md

```diff
@@ -15,6 +15,22 @@ You can use `promtool` to test your rules.
 ./promtool test rules test1.yml test2.yml test3.yml
 ```
 
+## Time within tests
```

Comment:
This section references `alert_test_case`, etc so it should go further down, as there's no reason for it to be before the things it references. I'd put it just before `Example`.

Reply:
I've pushed a commit that moves the section at the end of the file, in the Example section.